packages/browser/rollup.config.js

```diff
@@ -9,6 +9,14 @@ const commitHash = require('child_process')
   .trim();
 
 const terserInstance = terser({
+  compress: {
+    // Tell env.ts that we're building a browser bundle and that we do not
+    // want to have unnecessary debug functionality.
+    global_defs: {
+      __SENTRY_BROWSER_BUNDLE__: true,
+      __SENTRY_NDEBUG__: true,
```

Comment:
Is there a reason to make this the negative? Seems like it might be easy to miss the `N`, and IMHO the positive is easier to reason about.

Reply:
Since the default is `undefined` this is the only way I could make it work with the intended default behavior.

Reply:
Ah, okay, makes sense. What would you think of `__SENTRY_NO_DEBUG__`?

Reply:
Changed